src/components/SideMenu.jsx

```diff
@@ -0,0 +1,119 @@
+'use client'
+
+import { useState } from 'react'
+import {Roboto}from 'next/font/google'
+import Link from 'next/link'
+import {
+    Category,
+    Card,
+    ArrangeHorizontal,
+    TickCircle,
+    EmptyWallet,
+    DocumentDownload,
+    Star1,
+    Setting,
+    ArrowDown2,
+    ArrowUp2,
+    MessageQuestion,
+    CardReceive,
+    CardSend
+
+} from 'iconsax-react'
+
+const Logo = ({ src, alt }) => {
+    return (
+        <div className='mb-10'>
+            <img src={src} alt={alt} />
+        </div>
+    )
+}
+
+const MenuLink = ({ Icon, href, text, className }) => {
+    return (
+        <Link href={href} className={`flex items-center rounded-md w-full p-2 my-6 ${(className || "")}`}>
+            <Icon size={20} />
+            <span className="ml-6 flex-1 text-base">{text}</span>
+        </Link>
+    )
+}
+
+const MenuLinkGroup = ({ Icon, href, text, children }) => {
+    const [dropDownOn, setDropDownOn] = useState(false)
+
+    const dropDown = () => {
+        setDropDownOn(!dropDownOn)
+    }
+
+    return (
+        <div>
+            <div onClick={dropDown} className="flex items-center rounded-md border bg-yellow-500 pr-2">
```

Comment:
Can you import from next/navigation the pathname to set the current page active color.

Reply:
Reviewed by: @zutin